src/core/Akka.API.Tests/verify/DefaultLogFormatSpec.ShouldUseDefaultLogFormat.Net.verified.txt

```diff
@@ -0,0 +1,4 @@
+﻿[DEBUG][DateTime][Thread 0001][ActorSystem(test)] This is a test 1 cheese
+[INFO][DateTime][Thread 0001][ActorSystem(test)] This is a test 1
+[WARNING][DateTime][Thread 0001][ActorSystem(test)] This is a test 1
+[ERROR][DateTime][Thread 0001][ActorSystem(test)] This is a test 1
```

Comment:
No idea why the stack trace logs don't show up on .NET Framework, but they do on .NET 8